Source/inv.cpp

```diff
@@ -1942,13 +1942,13 @@ void ConsumeScroll(Player &player)
 	int itemSlot = player.executedSpell.spellFrom;
 	int itemIndex = 0;
 	Item *item;

```

Comment:
```suggestion
	Item *item = nullptr;
```

Reply:
Then we have to check `nullptr` later in the function.
Previous:
`if (!item->isEmpty() && isCurrentSpell(*item)) {`
Changed:
`if (item != nullptr && !item->isEmpty() && isCurrentSpell(*item)) {`